dev-requirements.txt

```diff
@@ -20,7 +20,7 @@ python-coveralls>=2.8.0; python_version == '2.7'
 
 # Unit test (no imports, invoked via py.test script)
 pytest>=3.3.0
-pytest-cov>=2.4.0
+pytest-cov<2.6
```

Comment:
It should still require the minimum, i.e. `pytest-cov>=2.4.0,<2.6`. Also, dd a TODO comment to not forget removing the pinning again once the issue is resolved.